src/main/webapp/app/exercises/text/manage/example-text-submission/example-text-submission.component.ts

```diff
@@ -260,25 +260,44 @@ export class ExampleTextSubmissionComponent extends TextAssessmentBaseComponent
             return;
         }
         this.tutorParticipationService.assessExampleSubmission(this.exampleSubmissionForNetwork(), this.exerciseId).subscribe(
-            () => this.jhiAlertService.success('artemisApp.exampleSubmission.assessScore.success'),
+            () => {
+                this.markAllFeedbackToCorrect();
+                this.jhiAlertService.success('artemisApp.exampleSubmission.assessScore.success');
+            },
             (error: HttpErrorResponse) => {
                 const errorType = error.headers.get('x-artemisapp-error');
 
-                switch (errorType) {
-                    case 'error.tooLow':
-                        this.jhiAlertService.error('artemisApp.exampleSubmission.assessScore.tooLow');
-                        break;
-                    case 'error.tooHigh':
-                        this.jhiAlertService.error('artemisApp.exampleSubmission.assessScore.tooHigh');
-                        break;
-                    default:
-                        onError(this.jhiAlertService, error);
-                        break;
+                if (errorType === 'error.invalid_assessment') {
+                    this.markAllFeedbackToCorrect();
+
+                    // Mark all wrongly made feedbacks accordingly.
+                    const correctionErrors: FeedbackCorrectionError[] = JSON.parse(error['error']['title'])['errors'];
+                    correctionErrors.forEach((res) => {
```

Comment:
same as above, `res` is a bit misleading

Reply:
Renamed it in the extracted class.